mkosi/__init__.py

```diff
@@ -6240,28 +6227,37 @@ def setup_network_veth(args: CommandLineArguments, root: str, do_run_build_scrip
         run(["systemctl", "--root", root, "enable", "systemd-networkd"])
 
 
+class BuildOutput(NamedTuple):
+    raw: Optional[BinaryIO]
+    archive: Optional[BinaryIO]
+    root_hash: Optional[str]
+    sshkey: Optional[TextIO]
+    split_root: Optional[BinaryIO]
+    split_verity: Optional[BinaryIO]
+    split_kernel: Optional[BinaryIO]
+
+    def raw_name(self) -> Optional[str]:
+        return self.raw.name if self.raw is not None else None
+
+    @classmethod
+    def Empty(cls) -> "BuildOutput":
```

Comment:
```suggestion
    def empty(cls) -> "BuildOutput":
```

Reply:
I assumed that the convention is to use class-like naming for "alternate constructors". But looking online, there seems to be no such rule ;)